stack/Stack.cs

```diff
@@ -58,92 +49,9 @@ public void Push(T data)
     // Removes the element at the top of the stack
     public T Pop()
     {
-        if (IsEmpty())
-            return default(T);
-
-        T element = stack.Last();
-
+        T t = stack.Last();
         stack.RemoveLast();
-
-        return element;
-    }
-
-    // Returns true if the stack is empty
-    public bool IsEmpty()
-    {
-        return (!stack.Any());
-    }
-
-
-    // Returns the last element of the stack
-    public T Peek()
-    {
-        if (IsEmpty())
-            return default(T);
-
-        return stack.Last();
-    }
-
-    /**
-    * gets the n top elements and rotates them
-    *
-    * if left == false then it will move the last element to behind the nth element(from the top)
-    *
-    * if left == true it will move the nth element(from the top) to the top
-    */
-    public void Rotate(int n, bool left)
```

Comment:
The issue mentions that this method should use stack operations to rotate the numbers. But you have removed the method here, which is not expected. Please use stack methods to accomplish this method.

Reply:
Ah ok, I misunderstood the ticket.  I will update my PR.